tarantool/dbapi.py

```diff
@@ -0,0 +1,227 @@
+# -*- coding: utf-8 -*-
+from tarantool.connection import Connection as BaseConnection
+from tarantool.error import *
+
+
+paramstyle = 'named'
+apilevel = "2.0"
+threadsafety = 1
+
+
+class Cursor:
+
+    def __init__(self, conn):
+        self._c = conn
+        self._lastrowid = None
+        self._rowcount = None
+        self.arraysize = 1
+        self._rows = None
+
+    def callproc(self, procname, *params):
+        """
+        Call a stored database procedure with the given name. The sequence of
+        parameters must contain one entry for each argument that the
+        procedure expects. The result of the call is returned as modified
+        copy of the input sequence. Input parameters are left untouched,
+        output and input/output parameters replaced with possibly new values.
+        """
+        raise NotSupportedError("callproc() method is not supported")
+
+    @property
+    def rows(self):
+        return self._rows
+
+    @property
+    def description(self):
+        # FIXME Implement this method please
+        raise NotImplementedError("callproc() method is not implemented")
+
+    def close(self):
+        """
+        Close the cursor now (rather than whenever __del__ is called).
+        The cursor will be unusable from this point forward; DatabaseError
+        exception will be raised if any operation is attempted with
+        the cursor.
+        """
+        self._c = None
+        self._rows = None
+
+    def _check_not_closed(self, error=None):
+        if self._c is None:
+            raise ProgrammingError(error or "Can not operate on closed cursor")
+
+    def execute(self, query, params=None):
+        """
+        Prepare and execute a database operation (query or command).
+        """
+        self._check_not_closed("Can not execute on closed cursor.")
+
+        response = self._c.execute(query, params)
+
+        self._rows = response.rows
+        self._rowcount = response.rowcount
```

Comment:
replaced to:
```
        if response.autoincrement_ids:
            self._lastrowid = response.autoincrement_ids[-1]
        else:
            self._lastrowid = -1
```
because rowcound has been changed to autoincrement_ids with the different return value (it was Integer, not it is a list)